components/raftstore/src/store/snap.rs

```diff
@@ -150,6 +150,10 @@ impl SnapKey {
             snap,
         ))
     }
+
+    pub fn get_snapshot_recv_path(&self) -> String {
```

Comment:
Why the change? V2 should not use v1's snapshot.

Reply:
Also it looks that get_snapshot_recv_path is not called at all.